kratos/geometries/geometry.h

```diff
@@ -143,6 +143,11 @@ class Geometry
     */
     typedef std::size_t SizeType;
 
+    /** Geometry shall behave similiar as indexed objects.
+    *   To allow query within data bases this type def is required.
+    */
+    typedef std::size_t result_type;
```

Comment:
I would use the `IndexType`

Reply:
I removed result_type for the moment as discussed, personally. If someone will need it for specific queries it can be added.